packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/setup-provider.tsx

```diff
@@ -289,10 +291,47 @@ Could you make a new website based on these notes and send back just the html fi
   });
 
   AIProvider.provide('createSlides', options => {
+    const SlideSchema = z.object({
+      page: z.number(),
+      type: z.enum(['name', 'title', 'content']),
+      content: z.string(),
+    });
+    type Slide = z.infer<typeof SlideSchema>;
+    const parseJson = (json: string) => {
+      try {
+        return SlideSchema.parse(JSON.parse(json));
+      } catch {
+        return null;
+      }
+    };
+    const postfix = (text: string): string => {
+      const slides = text
+        .split('\n')
+        .map(parseJson)
+        .filter((v): v is Slide => !!v);
+      return slides
+        .map(slide => {
```

Comment:
how about converting in backend?

Reply:
backend's workflow not impl a custom code node to allow this behaviour, i mark a todo here, will impl it after i made the yocto binding